src/Makefile.am

```diff
@@ -25,5 +25,5 @@ mtk.h protrack.h rad.h raw.h s3m.h sa2.h sng.h u6m.h player.h fmc.h mad.h \
 xad.h bmf.h flash.h hyp.h psi.h rat.h hybrid.h rol.h adtrack.h cff.h dtm.h \
 dmo.h fprovide.h database.h players.h xsm.h adlibemu.h kemuopl.h dro.h \
 realopl.h analopl.h temuopl.h msc.h rix.h adl.h jbm.h cmf.h surroundopl.h \
-dro2.h got.h cmflinel.h version.h wemuopl.h woodyopl.h nemuopl.h nukedopl.h adlib.h \
```

Comment:
I forgot to mention that AdPlug has MS-DOS version called AdPlay, so better to limit file names to 8.3 scheme (8 chars for name and 3 chars for extension).